content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js

```diff
@@ -1683,18 +1683,18 @@ describe("GIVEN createYamlString function", () => {
           enabled: false
         nodeSelector: false
       csireverseproxy:
-        image: dellemc/csipowermax-reverseproxy:v2.5.0
+        image: quay.io/dell/container-storage-modules/csipowermax-reverseproxy:v2.5.0
```

Comment:
Are we using the latest images version? csipowermax-reverseproxy:v2.5.0 it should be csipowermax-reverseproxy:v2.11.0